src/cpp_dl_benchmark/onnxruntime_launcher/src/onnxruntime_launcher.cpp

```diff
@@ -181,3 +191,23 @@ int ONNXLauncher::evaluate(int iterations_num, uint64_t time_limit_ns) {
 
     return iteration;
 }
+
+void ONNXLauncher::dump_output() {
+    std::vector<std::pair<const float*, size_t>> tmp;
+    for(int i = 0; i < tensors.size();i++){
+        std::string name = "output" + std::to_string(i);
+        std::ofstream file(name);
+        auto Tens = run_for_output(tensors[i]);
```

Comment:
лучше придерживаться именования переменных с маленькой буквы
```suggestion
        auto tens = run_for_output(tensors[i]);
```